build/source/driver/multi_driver.f90

```diff
@@ -873,19 +966,39 @@ program multi_driver
                   ! error control
                   err,message)                ! intent(out):   error control
 
-  ! error control
+  !----- save timing information ------------------------------------------------
+
+  !$omp critical(saveTiming)
+
+  ! check errors
   call handle_err(err,message)
 
+  ! save timing information
+  timeGRU(iGRU)          = omp_get_wtime() - timeGRUstart(iGRU)
+  timeGRUcompleted(iGRU) = omp_get_wtime() - openMPstart
```

Comment:
nit. The use of spaces is inconsistent in this PR. I see lines with no space around `=`, aligned `=`, and 1 space on either side of ` = `. Does the SUMMA style guide specify how this should be?

Reply:
No - https://github.com/NCAR/summa/blob/master/docs/development/SUMMA_coding_conventions.md